locales/en/messages.json

```diff
@@ -4675,6 +4678,211 @@
+        "description": "Help text for the pit mode field of the VTX tab"
+    },
+    "vtxPitModeFrequency": {
+        "message": "Pit Mode frequency",
+        "description": "Text of one of the fields of the VTX tab"
+    },
+    "vtxPitModeFrequencyHelp": {
+        "message": "Frequency at which the Pit Mode changes when enabled.",
+        "description": "Help text for the pit mode field of the VTX tab"
+    },
+    "vtxLowPowerDisarm": {
+        "message": "Low Power Disarm",
+        "description": "Text of one of the fields of the VTX tab"
+    },
+    "vtxLowPowerDisarmHelp": {
+        "message": "When enabled, the VTX uses the lowest available power when disarmed (except if a failsafe has occurred).",
+        "description": "Help text for the low power disarm field of the VTX tab"
+    },
+    "vtxLowPowerDisarmOption_0": {
+        "message": "Off",
+        "description": "One of the options for the Low Power Disarm mode of the VTX"
+    },
+    "vtxLowPowerDisarmOption_1": {
+        "message": "On",
+        "description": "One of the options for the Low Power Disarm mode of the VTX"
+    },
+    "vtxLowPowerDisarmOption_2": {
+        "message": "On until first arm",
+        "description": "One of the options for the Low Power Disarm mode of the VTX"
+    },
+    "vtxOptionYes": {
+        "message": "Yes",
+        "description": "Used in several descriptions in the VTX tab (device ready, pit mode, low power disarm...)"
+    },
+    "vtxOptionNo": {
+        "message": "No",
+        "description": "Used in several descriptions in the VTX tab (device ready, pit mode, low power disarm...)"
+    },
+    "vtxTable": {
+        "message": "VTX Table",
+        "description": "Text of the header of the VTX Table element in the VTX tab"
+    },
+    "vtxTablePowerLevels": {
+        "message": "Number of power levels",
+        "description": "Text of one of the fields of the VTX Table element in the VTX tab"
+    },
+    "vtxTablePowerLevelsHelp": {
+        "message": "This defines the number of power levels supported by your VTX",
+        "description": "Help for the number of power levels field of the VTX Table element in the VTX tab"
+    },
+    "vtxTablePowerLevelsTableHelp": {
+        "message": "This table represents the different values of power that can be used for the VTX. They are divided into two: <br><b>- $t(vtxTablePowerLevelsValue.message):</b> each power level requires a value that is defined by the hardware manufacturer. For example, for Tramp it is the same than the real output (25, 100, 200, 400...), SmartAudio 1.0 usually uses 7, 16, 25, 40 and RTC6705 and SmartAudio 2.0 uses a sequential (0, 1, 2, 3). <br><b>- $t(vtxTablePowerLevelsLabel.message):</b> you can put here the label you want for each power level value. It can be numbers (25, 200, 600, 1.2), letters (OFF, MIN, MAX) or a mix of them. <br><br>You must configure <b>only</b> the power levels that are legal at your contry.",
```

Comment:
I wouldn't put any actual values into the help text - there are now so many different VTX out there implementing the different protocols, we are almost guaranteed to be wrong for some and get bug reports.

Reply:
True, I didn't know it depends on the implementation. I will fix it.